MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -4717,6 +4752,15 @@ public static string GetHwTonemapFilter(EncodingOptions options, string hwTonema
             subFilters?.RemoveAll(filter => string.IsNullOrEmpty(filter));
             overlayFilters?.RemoveAll(filter => string.IsNullOrEmpty(filter));
 
+            var framerate = GetFramerateParam(state);
+            if (framerate.HasValue)
+            {
+                mainFilters.Insert(0, string.Format(
+                    CultureInfo.InvariantCulture,
+                    "fps={0}",
+                    framerate.Value));
+            }
```

Comment:
The actual fps can affect the key frame arguments. We may need some changes in `GetHlsVideoKeyFrameArguments()`.

https://github.com/jellyfin/jellyfin/blob/af5184f84170bde0d66d89be2b953f950c7dfc86/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs#L1401-L1413

Reply:
As mentioned in the original review, this could also potentially affect ```GetVideoQualityParam``` which uses the request Framerate/MaxFramerate to set the -r ffmpeg argument.